composer.json

```diff
@@ -60,8 +60,8 @@
         "config-plugin": {
             "params": "params.php",
             "bootstrap": "bootstrap.php",
-            "widgets": [],
-            "widgets-themes": []
+            "widgets": [ ],
+            "widgets-themes": [ ]
```

Comment:
Why?

Reply:
![image](https://github.com/yiisoft/widget/assets/67489841/f5ff59f3-8c9c-4b2e-8cc6-73c3ee92a459)

that's why. I disabled it now.